ui/admin-portal/src/app/services/auth.service.ts

```diff
@@ -149,11 +149,30 @@ export class AuthService {
   }
 
   isCandidateOurs(candidate: ShortCandidate): boolean {
-    return true; //todo
+    let ours = false;
```

Comment:
```suggestion
    let ours: boolean = false;
```

Reply:
In fact this is completely typesafe in Typescript, and recommended practice. I only add an explicit type occasionally by way of documentation where I think it is informative. 

Adding a line like ours = 123 after the declaration without the boolean, will result in a compile error.

A useful way of playing around with Typescript is the "playground" - here https://www.typescriptlang.org/play/index.html

Reply:
I wasn't sure if this might be overkill when I suggested it but thought it would be good to get your view. Good for me to know for future that it's recommended practice - thanks John.